zkevm-circuits/src/pi_circuit.rs

```diff
@@ -858,15 +884,118 @@ impl<F: Field> PiCircuitConfig<F> {
         Ok(rpi_bytes_keccakrlc_cell)
     }
 
+    /// Assigns a wd_table row and stores the values in a vec for the raw_public_inputs column
+    /// current_rpi_offset is used to record current rpi offset
+    #[allow(clippy::too_many_arguments)]
+    fn assign_wd_table_row(
+        &self,
+        region: &mut Region<'_, F>,
+        offset: usize,
+        wd: &Withdrawal,
+        rpi_bytes_keccak_rlc: &mut Value<F>,
+        challenges: &Challenges<Value<F>>,
+        current_rpi_offset: &mut usize,
+        rpi_bytes: &mut [u8],
+        zero_cell: AssignedCell<F, F>,
+    ) -> Result<(), Error> {
+        let id_assigned_cell = region.assign_advice(
+            || "withdrawal_id",
+            self.wd_table.id,
+            offset,
+            || Value::known(F::from(wd.id)),
+        )?;
+        let vid_assigned_cell = region.assign_advice(
+            || "validator_id",
+            self.wd_table.validator_id,
+            offset,
+            || Value::known(F::from(wd.validator_id)),
+        )?;
+        let address_word = Word::<F>::from(wd.address);
+        let addr_lo_assigned_cell = region.assign_advice(
+            || "address_lo",
+            self.wd_table.address.lo(),
+            offset,
+            || Value::known(address_word.lo()),
+        )?;
+        let addr_hi_assigned_cell = region.assign_advice(
+            || "address_hi",
+            self.wd_table.address.hi(),
+            offset,
+            || Value::known(address_word.hi()),
+        )?;
```

Comment:
A simplify version

```diff
-        let address_word = Word::<F>::from(wd.address);
-       let addr_lo_assigned_cell = region.assign_advice(
-            || "address_lo",
-            self.wd_table.address.lo(),
-            offset,
-            || Value::known(address_word.lo()),
-        )?;
-        let addr_hi_assigned_cell = region.assign_advice(
-            || "address_hi",
-            self.wd_table.address.hi(),
-            offset,
-            || Value::known(address_word.hi()),
-        )?;
+  Word::<F>::from(wd.address).into_value().assign_advice(region, || "address", self.wd_table.address, offset)?;
```

Reply:
fixed in [d616eda](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1634/commits/d616eda437f3316ec3468525fea605975b801897)